ipc/ipc_message_utils.cc

```diff
@@ -753,6 +757,19 @@ bool ParamTraits<base::SharedMemoryHandle>::Read(const base::Pickle* m,
   *r = base::SharedMemoryHandle(mach_port_mac.get_mach_port(),
                                 static_cast<size_t>(size), guid);
 #elif defined(OS_POSIX)
+#if defined(CASTANETS)
+  if (static_cast<internal::PlatformFileAttachment*>(attachment.get())->file()
+      == -1) {
```

Comment:
[Q] Is there a possibility for attachment.get() to return NULL? If there is, attachment.get()->file() might cause seg fault.

Reply:
Could kCastanetsHandle defined in mojo/public/c/system/platform_handle.h be used instead of -1?

Reply:
@Chikdolman No, it's not possible to access out of bound of an IPC message in this method.
@joonghunpark To add #include mojo/public/c/system/platform_handle.h is needed for that. I think kCastanetsHandle would be better to move to shared_memory_helper.h or some other file. I have no idea where is best. Why don't we just open a new PR for that?